week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-panel.component.ts

```diff
@@ -0,0 +1,25 @@
+import { article, p } from '../framework/dom-creators.js';
+import cartItemSum from './cart-item-sum.component.js';
+import cartItem from './cart-item.component.js';
+import type Item from '../model/item';
+
+interface ICartPanelProps {
+  items: Item[];
+  heading?: string;
+}
```

Comment:
Byłbym nawet bardziej radykalny i nazwał to po prostu `PropTypes` - bo takie w tych komponentach pełni to funkcje.
I pokazuje kształt danych oczekiwanych per `component`.

Reply:
👍🏻